frontend/packages/kubevirt-plugin/src/components/vms/vm-details-page.tsx

```diff
@@ -41,6 +49,7 @@ export const breadcrumbsForVMPage = (match: any) => () => [
 
 export const VirtualMachinesDetailsPage: React.FC<VirtualMachinesDetailsPageProps> = (props) => {
   const { name, ns: namespace } = props.match.params;
+  const isSnapshotsAvailable = useModelLoaded(VirtualMachineSnapshotModel.kind);
```

Comment:
shouldn't `referenceForModel` be used here?

Reply:
depends on the use case, some would use `referenceForModel(model)` but in our case the resource in store is called `VMSnapshotModel` so we need `VMSnapshotModel.kind`

Reply:
it might get loaded only with the group+version to the store so we should use `referenceForModel` to be safe